source/system/Demo_User_Guides/TI_Apps_Launcher_User_Guide.rst

```diff
@@ -365,14 +365,70 @@ _________________________
 
 3. Once you enter the necessary values for both HTTPS and No proxy, click on `Set Proxy` to set the proxy configuration.
 
+
+Using terminal
+______________
+
+.. ifconfig:: CONFIG_sdk in ('SITARA')
+
+    The terminal button on the left panel can be used to open a terminal application, which is
+    based on QMLtermwidget.
+
+    .. Image:: /images/terminal.png
+        :height: 400
+
+.. ifconfig:: CONFIG_sdk not in ('SITARA')
+
+   This section is not valid for this platform.
+
+Using Wi-Fi Demo
+________________
+
+.. ifconfig:: CONFIG_part_variant in ('AM62X', 'AM62PX')
+
+    The *Wifi* button on the left panel allows you to connect to an external Wi-Fi network.
+
+    Ensure the required overlay is applied and the WiFi card is connected to SK-EVM. For more
+    information, refer to :ref:`enable_m2cc3301`.
+
+    Click the toggle button to turn on Wi-Fi.
+
+    .. Image:: /images/wifi_home.png
+        :height: 400
+
+    * Click on the *Refresh* button to scan for available SSIDs.
+    * Select an SSID from the list, then enter the passphrase and Wi-Fi security type.
+    * For certain security types, you may need to provide additional details.
+    * After entering all the required info, click on *Connect*.
+
+    .. Image:: /images/wifi_ssid.png
+        :height: 400
+
+    If the all entered details are correct, device will be connected to the selected Wi-Fi
+    network. If needed, click *Disconnect* to disconnect from the network and return to
+    SSID selection menu.
+
+    .. Image:: /images/wifi_connected.png
+        :height: 400
+
+    .. ifconfig:: CONFIG_part_variant in ('AM62X')
```

Comment:
Indentation for all ifcongig, note, Image directive doesn’t follow the CONTRIBUTING guidelines- https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace